pandas/tests/generic/test_generic.py

```diff
@@ -897,3 +897,13 @@ def test_flags_identity(self, as_frame):
         assert s.flags is s.flags
         s2 = s.copy()
         assert s2.flags is not s.flags
+
+    def test_slice_incomplete_datetimes(self):
```

Comment:
I'd call it `test_slice_irregular_datetime_index_with_nan`

Reply:
done.